CI/linux/01_install_dependencies.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+##############################################################################
+# Linux dependency management function
+##############################################################################
+#
+# This script file can be included in build scripts for Linux or run directly
+# with the -s/--standalone switch
+#
+##############################################################################
+
+# Halt on errors
+set -eE
+
+install_build-deps() {
+    shift
+    status "Install OBS build dependencies"
+    trap "caught_error 'install_build-deps'" ERR
+
+    sudo apt-get install -y $@
+}
+
+install_obs-deps() {
+    shift
+    status "Install OBS dependencies"
+    trap "caught_error 'install_obs-deps'" ERR
+
+    if [ -z "${DISABLE_PIPEWIRE}" ]; then
+        sudo apt-get install -y $@ libpipewire-0.3-dev
+    else
+        sudo apt-get install -y $@
+    fi
+}
+
+install_qt-deps() {
+    shift
+    status "Install Qt dependencies"
+    trap "caught_error 'install_qt-deps'" ERR
+
+    sudo apt-get install -y $@
+}
+
+install_cef() {
+    shift
+    status "Setup for dependency CEF v${1}"
+    ensure_dir "${DEPS_BUILD_DIR}"
+
+    if [ "${CI}" -a "${RESTORED_CEF}" ]; then
+        _SKIP=TRUE
+    elif [ -d "${DEPS_BUILD_DIR}/cef_binary_${1}_linux64" -a -f "${DEPS_BUILD_DIR}/cef_binary_${1}_linux64/build/libcef_dll_wrapper/libcef_dll_wrapper.a" ]; then
+        _SKIP=TRUE
+    fi
+
+    if [ -z "${_SKIP}" ]; then
+        step "Download..."
+        ${CURLCMD:-curl -O} https://cdn-fastly.obsproject.com/downloads/cef_binary_${1}_linux64.tar.bz2
+        step "Unpack..."
+        /usr/bin/tar -xf cef_binary_${1}_linux64.tar.bz2
+    else
+        step "Found existing CEF Framework and loader library..."
+    fi
+}
+
+install_plugin-deps() {
+    shift
+    status "Install plugin dependencies"
+    trap "caught_error 'install_plugin-deps'" ERR
+
+    sudo apt-get install -y $@
+}
+
+install_dependencies() {
+    status "Set up apt"
+    trap "caught_error 'install_dependencies'" ERR
+
+    BUILD_DEPS=(
+        "build-deps cmake ninja-build pkg-config clang clang-format build-essential curl ccache"
+        "obs-deps libavcodec-dev libavdevice-dev libavfilter-dev libavformat-dev libavutil-dev libswresample-dev \
+         libswscale-dev libx264-dev libcurl4-openssl-dev libmbedtls-dev libgl1-mesa-dev libjansson-dev \
+         libluajit-5.1-dev python3-dev libx11-dev libxcb-randr0-dev libxcb-shm0-dev libxcb-xinerama0-dev \
+         libxcomposite-dev libxinerama-dev libxcb1-dev libx11-xcb-dev libxcb-xfixes0-dev swig libcmocka-dev \
+         libxss-dev libglvnd-dev libgles2-mesa libgles2-mesa-dev libwayland-dev"
+        "qt-deps qtbase5-dev qtbase5-private-dev libqt5svg5-dev libqt5x11extras5-dev qtwayland5"
```

Comment:
Nitpicking: Qt5X11Extras still put as build deps in this script

Reply:
Yep, left it in there to check if y'all paid attention.. 👀